docs/guides/tools.md

```diff
@@ -42,3 +42,66 @@ type Query {
   myName: String!
 }
 ```
+
+---
+
+### `depth_limit_validator`
+
+Create a validator to limit the complexity of queries by their depth to protect against malicious
+queries.
+
+```python
+from graphql import ValidationRule
+
+class DepthLimitOptions(TypedDict):
```

Comment:
we wanted to use a class for configuration, should we use the same approach here?

Reply:
Actually, since there is only 1 option, I'm just going to inline it.